docs/docs/configuration/license_plate_recognition.md

```diff
@@ -51,7 +51,7 @@ Fine-tune the LPR feature using these optional parameters:
 
 - **`detection_threshold`**: License plate object detection confidence score required before recognition runs.
   - Default: `0.7`
-  - Note: If you are using a Frigate+ model and you set the `threshold` in your objects config for `license_plate` higher than this value, recognition will never run. It's best to ensure these values match, or this `detection_threshold` is lower than your object config `threshold`.
+  - Note: If you are using a Frigate+ model and you set the `min_score` in your objects config for `license_plate` higher than this value, recognition will never run. It's best to ensure these values match, or this `detection_threshold` is lower than your object config `min_score`.
```

Comment:
Your change is true, but it obfuscates the reason why I included the comment initially. The original statement could be clarified if we want to say something about `min_score`. If a user sets their `license_plate` threshold in their object config lower than lpr's `detect_threshold`, recognition will never run, because Frigate itself will never pass the license plate to the detector. The `detection_threshold` is a secondary (and perhaps unnecessary) pass through for Frigate+ users. I made this note because some users may be using the yolov9 model and have no other config for a threshold value. 

With whatever we decide, we should probably make it consistent for both lpr and face rec.

Reply:
`threshold` is not used anywhere for attributes or sub labels like fedex, amazon. The only field that matters for that is min_score, since threshold is only ever used to consider an object to be a true or false positive. You could set threshold to 0.0 and license_plates would still be detected as only min_score matters for that. See the below code

https://github.com/blakeblackshear/frigate/blob/d2368f5cbbbf6b08b68853aff01b5b6e26109a27/frigate/config/config.py#L422-L427

Reply:
Right, sorry, I could have said it better.

In LPR, for the yolov9 model, `detection_threshold` is used:

https://github.com/blakeblackshear/frigate/blob/d2368f5cbbbf6b08b68853aff01b5b6e26109a27/frigate/data_processing/common/license_plate/mixin.py#L708

Anything below that threshold is filtered out, so this could probably be removed altogether:

https://github.com/blakeblackshear/frigate/blob/d2368f5cbbbf6b08b68853aff01b5b6e26109a27/frigate/data_processing/common/license_plate/mixin.py#L940

And then maybe another comment could be added that `detection_threshold` is only used when NOT using a Frigate+ model or any other model that detects `license_plate`.

Reply:
right, that is how face recognition works currently, I will update

Reply:
perfect